recipe/meta.yaml

```diff
@@ -11,36 +12,21 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win and vc<14]
 
-test:
-  requires:
-    - clangdev
-    - {{ compiler('cxx') }}  # [unix]
-    - vs2017_{{ target_platform }}  # [win]
-  commands:
-    - if not exist %LIBRARY_BIN%\\libomp.dll exit 1   # [win]
-    - test -f $PREFIX/lib/libomp.so                   # [linux]
-    - test -f $PREFIX/lib/libomp.dylib                # [osx]
-    - if not exist %LIBRARY_LIB%\\libomp.lib exit 1   # [win]
-    - if not exist %LIBRARY_INC%\\omp.h exit 1        # [win]
-    - test -f $PREFIX/include/omp.h                   # [unix]
-    - $PREFIX/bin/clang -v -fopenmp -I$PREFIX/include -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib omp_hello.c -o omp_hello  # [not win]
-    - "%LIBRARY_BIN%\\clang -v -fopenmp -I%LIBRARY_INC% -L%LIBRARY_LIB% omp_hello.c -o omp_hello.exe"  # [win]
-    - ./omp_hello            # [unix]
-    - "%cd%\\omp_hello.exe"  # [win]
-  files:
-    - omp_hello.c
-
 outputs:
   - name: llvm-openmp
     script: build-llvm-openmp.sh  # [not win]
     script: bld-llvm-openmp.bat   # [win]
     build:
+      ignore_run_exports:
+        - libgcc-ng
       run_exports:
         strong:
-          - {{ pin_subpackage("llvm-openmp", max_pin=None) }}
+          - {{ pin_subpackage("llvm-openmp", max_pin=None) }}  # [not linux]
+          - openmp_impl >={{ openmp_ver }}  # [linux]
+          - openmp_impl {{ openmp_ver }} 0_llvm            # [linux]
```

Comment:
@isuruf This set of selectors does not make sense to me. The second appears to override the first. Did you mean to pin the openmp version here?